docs/quickstarts/ratify-cli.md

```diff
@@ -0,0 +1,119 @@
+# Ratify Cli
```

Comment:
```suggestion
# Ratify CLI
```

Reply:
CLI is acronym of **Command Line Interface** so it's recommended to use upper case